website/docs/cli/commands/init.html.md

```diff
@@ -2,8 +2,7 @@
 layout: "docs"
 page_title: "Command: init"
 sidebar_current: "docs-commands-init"
-description: |-
-  The `terraform init` command is used to initialize a Terraform configuration. This is the first command that should be run for any new or existing Terraform configuration. It is safe to run this command multiple times.
+description: "terraform init initializes a working directory containing Terraform configuration files."
```

Comment:
for this one we might want to say something about installing providers too :)

Reply:
>